src/module-info.java

```diff
@@ -7,6 +7,11 @@
     requires aion.crypto;
     requires logback.core;
     requires logback.classic;
+    requires jdk.sctp;
```

Comment:
why api need this module?

Reply:
It's used by https://github.com/aionnetwork/aion_api/blob/master/src/org/aion/api/impl/Admin.java#L29

Reply:
I see, can you change the exception type with the java.base, then we can remove this package import&export.